Migrating urlresolver topic to queries subdir as per Magedoc3977 (#5084)
magento/devdocs: guides/v2.3/graphql/reference/products.md

`default` | String | The default sort field
`options` | `SortField` | An array that contains all the fields you can use for sorting

### SortField object

Attribute | Type | Description
--- | --- | ---
`label` | String | The attribute's label
`value` | String | The attribute name or code to use as the sort field

### Website object {#websiteObject}

> Use the `Website` attributes to retrieve information about the website's configuration; such as, website name, website code, and default group id.

Reviewer: ```suggestion
Use the `Website` attributes to retrieve information about the website's configuration, which includes the website name, website code, and default group ID.
```

Author: fixed
